### ISSUE TITLE
Last release not working

### ISSUE THREAD

**Issue by MissiaL** [kiwicom/schemathesis#338]
When I ran test from my code i got errors
```
  File "/usr/local/lib/python3.7/site-packages/_pytest/assertion/rewrite.py", line 143, in exec_module
    exec(co, module.__dict__)
  File "/usr/local/lib/python3.7/site-packages/schemathesis/__init__.py", line 7, in <module>
    init_default_strategies()
  File "/usr/local/lib/python3.7/site-packages/schemathesis/_hypothesis.py", line 187, in init_default_strategies
    register_string_format("binary", st.binary())
  File "/usr/local/lib/python3.7/site-packages/schemathesis/_hypothesis.py", line 181, in register_string_format
    from hypothesis_jsonschema._impl import STRING_FORMATS  # pylint: disable=import-outside-toplevel
ModuleNotFoundError: No module named 'hypothesis_jsonschema._impl'
```

**Comment by Stranger6667** [comment]
Yep, it does not yet support `hypothesis-jsonschema>=0.9.13` - #337 should solve it

**Comment by Stranger6667** [comment]
Fixed in `0.20.5` release - https://pypi.org/project/schemathesis/0.20.5/
